lib/pages/openedListPage.dart

```diff
@@ -0,0 +1,135 @@
+import 'package:MobileOne/localization/localization.dart';
+import 'package:MobileOne/pages/widget_item.dart';
+import 'package:MobileOne/pages/widget_popup.dart';
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+import '../localization/localization.dart';
+
+const Color GREEN = Colors.green;
+const Color GREY = Colors.grey;
+const Color GREY600 = Colors.grey;
+const Color RED = Colors.red;
+const Color WHITE = Colors.white;
+const Color TRANSPARENT = Colors.transparent;
+
+final itemNameController = new TextEditingController();
+final itemCountController = new TextEditingController();
```

Comment:
why are you setting those two controllers in the OpenedListPage ? They are only used in the WidgetPopup. Please move thos two controllers in the WidgetPopup widget